config/kibana.yml

```diff
@@ -102,3 +102,36 @@
 # The default locale. This locale can be used in certain circumstances to substitute any missing
 # translations.
 #i18n.defaultLocale: "en"
+
+logging:
+  appenders:
+    console:
+      kind: console
+      pattern: "[{timestamp}][{level}][{context}] {message}"
+    server-console:
+      kind: console
+    file:
+      kind: file
+      path: path
+    rolling-file-by-size:
+      kind: rolling-file
+      path: path
+      policy:
+        kind: time
+        limit: 30s
+    rolling-file-by-time:
+      kind: rolling-file
```

Comment:
No, it's not that advanced right now and we don't really need to separate them. It makes sense to start from single file-based appender and eventually split them if we need to.